src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -2,84 +2,41 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*=========================================================================
-**
-** Purpose: 
-** A complex number z is a number of the form z = x + yi, where x and y 
-** are real numbers, and i is the imaginary unit, with the property i2= -1.
-**
-===========================================================================*/
-
-using System;
 using System.Globalization;
 using System.Diagnostics.CodeAnalysis;
-using System.Diagnostics.Contracts;
 
 namespace System.Numerics
 {
+    /// <summary>
+    /// A complex number z is a number of the form z = x + yi, where x and y 
+    /// are real numbers, and i is the imaginary unit, with the property i2= -1.
+    /// </summary>
```

Comment:
Comment came from the massive file header at the top (that I removed to combine with the class summary)

Sorry about the doc changes - It's probably not worth the bother to undo that but I'll keep it in mind for the future